File: Python/D6 - T/1 - while loop/1.py
#  while(condition):
    # body of loop

# Write a program to do addition of numbers 0 to 5.

n = 0
addUpto = int(input('Enter the number: '))
ans = 0

while(n <= addUpto):
    ans = ans + n
    n = n + 1

print(ans)

File: C/D2/1 - max number/1.cpp
// Write a C program to find the max number from 3 numbers.

#include <stdio.h>

int main()
{
    int a, b, c;

    printf("Enter values : ");
    scanf("%d %d %d", &a, &b, &c);

    if (a > b && a > c)
    {
        printf("Max number is %d\n", a);
    }
    else if (b > c && b > a)
    {
        printf("Max number is %d\n", b);
    }
    else
    {
        printf("Max number is %d\n", c);
    }
}
